pygments/styles/staroffice.py

```diff
@@ -0,0 +1,28 @@
+"""

```

Comment:
You need to add this style to the `STYLE_MAP` in `pygments/styles/__init__.py`. It won't be found without this. (Yes, it's inconsistent that we don't use an autogenerated mapping file for that like lexers and formatters.)